bazel/repository_locations.bzl

```diff
@@ -64,9 +64,10 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/circonus-labs/libcircllhist/archive/63a16dd6f2fc7bc841bb17ff92be8318df60e2e1.tar.gz"],
     ),
     com_github_cyan4973_xxhash = dict(
-        sha256 = "b34792646d5e19964bb7bba24f06cb13aecaac623ab91a54da08aa19d3686d7e",
-        strip_prefix = "xxHash-0.7.0",
-        urls = ["https://github.com/Cyan4973/xxHash/archive/v0.7.0.tar.gz"],
+        sha256 = "77876fa31d3132c8c22b81cb219eac5b1f035637d5ca48bad617380043e21ffd",
+        strip_prefix = "xxHash-6137bfc606e08c791fccab835928de7e5ed41308",
+        # 2019-07-11
+        urls = ["https://github.com/Cyan4973/xxHash/archive/6137bfc606e08c791fccab835928de7e5ed41308.tar.gz"],
```

Comment:
use 0.7.1? I think that contains this commit

Reply:
Agreed, and confirmed.